src/base/base.c

```diff
@@ -874,18 +884,36 @@ static bool should_run_user_handler_wrapper(zend_execute_data *execute_data)
 #endif
 }
 
+void xdebug_save_statement_handler(zend_extension *extension, statement_handler_func_t statement_handler)
```

Comment:
This function is used to save a pointer to the xdebug zend_extension and to the xdebug statement handler so that we can use this data to enable or disable this handler